composer.json

```diff
@@ -39,9 +39,13 @@
         "symfony/validator": "~2.7|~3.0|~4.0",
         "symfony/property-info": "~2.8|~3.0|~4.0",
         "symfony/phpunit-bridge": "~2.7|~3.0|~4.0",
-        "twig/twig": "~1.12|~2.0",
+        "twig/twig": "~1.26|~2.0",
```

Comment:
Increased to be able to call `Twig_Environment::addRuntimeLoader` in tests. Notice it's require-dev section